cities.json

```diff
@@ -2840,6 +2858,7 @@
   {"id": "2161212","name": "السوق الجديد","state_id": "21612"},
   {"id": "2161213","name": "السعيدة", "state_id":"21612"},
   {"id": "2161214","name": "الهيشرية", "state_id":"21612"},
+
   // For "Sifax" state, apparently there's 17 cities, but here are only 16 cities 
   // https://ar.wikipedia.org/wiki/%D9%88%D9%84%D8%A7%D9%8A%D8%A9_%D8%B5%D9%81%D8%A7%D9%82%D8%B3
```

Comment:
The link [here](https://ar.wikipedia.org/wiki/%D9%82%D8%A7%D8%A6%D9%85%D8%A9_%D9%88%D9%84%D8%A7%D9%8A%D8%A7%D8%AA_%D8%AA%D9%88%D9%86%D8%B3) clarifies that it only has 16 cities 
and they are listed [here](https://ar.wikipedia.org/wiki/%D8%AA%D8%B5%D9%86%D9%8A%D9%81:%D9%85%D8%B9%D8%AA%D9%85%D8%AF%D9%8A%D8%A7%D8%AA_%D9%88%D9%84%D8%A7%D9%8A%D8%A9_%D8%B5%D9%81%D8%A7%D9%82%D8%B3)
@Boudissa-Ihab

Reply:
Yes it's true, i think we'll go with 16 cities, i also found this [link](https://ar.wikipedia.org/wiki/%D8%A7%D9%84%D8%AA%D9%82%D8%B3%D9%8A%D9%85_%D8%A7%D9%84%D8%A5%D8%AF%D8%A7%D8%B1%D9%8A_%D9%81%D9%8A_%D8%AA%D9%88%D9%86%D8%B3) that mentions 16 cities for Sifax state (with all other states too)

Reply:
Nice, I did check both links and they all the same... now you can review Tunisia!